src/Indice.Features.Messages.App/src/app/features/distribution-lists/edit/contacts/distribution-list-contacts.component.ts

```diff
@@ -6,6 +6,7 @@ import { Observable, Subscription } from 'rxjs';
 import { map } from 'rxjs/operators';
 import { Contact, ContactResultSet, DistributionList, MessagesApiClient } from 'src/app/core/services/messages-api.service';
 import { BasicModalComponent } from 'src/app/shared/components/basic-modal/basic-modal.component';
+import { FileResponse } from '../../../../core/services/messages-api.service';
```

Comment:
I prefer absolute paths for imports from the src folder as above. These are easier for moving resources arround.